packages/maskbook/src/plugins/ITO/Worker/apis/index.ts

```diff
@@ -90,33 +90,39 @@ export async function getTradeInfo(pid: string, trader: string) {
             token: JSON_PayloadOutMask['token']
             amount: string
         }[]
-    }>(`
-    {
-        buyInfos (where: { pool: "${pid.toLowerCase()}", buyer: "${trader.toLowerCase()}" }) {
-            buyer {
-                ${TRADER_FIELDS}
-            }
-            token {
-                ${TOKEN_FIELDS}
-            }
-            amount
-            amount_sold
-            amount_bought
-        }
-        sellInfos (where: { pool: "${pid.toLowerCase()}", seller: "${trader.toLowerCase()}" }) {
-            seller {
-                address
-            }
-            amount
-        }
-        destructInfos (where: { pool: "${pid.toLowerCase()}", seller: "${trader.toLowerCase()}" }) {
-            seller {
-                address
+    }>({
+        query: gql`
```

Comment:
1. Instead of injecting `${pid.toLowerCase()}` to string template directly, declare it in `variables` option.
2. `gql` template tag provides highlight makes code more readable.

<img src="https://user-images.githubusercontent.com/63733714/129433871-06f4cf11-4c82-41db-8237-c2bffee1fb46.png" width=400 />

<img src="https://user-images.githubusercontent.com/63733714/129433864-ab05f0cc-a19e-43ff-9488-4a691333cf34.png" width=400 />